tests/Agent/UnitTests/CompositeTests/CrossAgentTests/SecurityPolicies/SecurityPoliciesCrossAgentTests.cs

```diff
@@ -58,9 +58,10 @@ public void Setup()
         }
 
         [TearDown]
-        public static void TearDown()
+        public void TearDown()
         {
             _compositeTestAgent.Dispose();
```

Comment:
The _compositeTestAgent field should probably not be static too.

Reply:
Good catch! Done.